### PR TITLE
change references from soc to dd academy

### PR DESCRIPTION
**Pull request by PBillingsby**

### CODE REVIEW

**Comment by PBillingsby** [comment]
Will keep this as draft until all references are removed. Still need to change repo name and update accordingly

File: components/WalletProvider.tsx [hunk 1]
```diff
@@ -16,6 +16,7 @@ const { chains, provider } = configureChains(
 )
 
 const { connectors } = getDefaultWallets({
+  // Change appName to D_D Academy
```
Comment (briangershon) [inline components/WalletProvider.tsx:19]: Should we make this change now?

**Review by briangershon — APPROVED** [review]
Thanks @PBillingsby for all the naming updates!

I just had one comment/question.

File: components/WalletProvider.tsx (hunk 1 above)
Comment (PBillingsby) [reply components/WalletProvider.tsx:19]: I left it considering we aren't using wallet connection just yet. Totally fine with changing it now!

Comment (briangershon) [reply components/WalletProvider.tsx:19]: Ah cool, either way works! Dealer's (Author's) choice! ☀️

Comment (PBillingsby) [reply components/WalletProvider.tsx]: Changed it.

**Pull request merged**